sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/test/scala/com/azure/cosmos/spark/SparkE2EThroughputControlSpec.scala

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.spark
+
+import com.azure.cosmos.implementation.TestConfigurations
+
+class SparkE2EThroughputControlSpec extends IntegrationSpec with Spark with CosmosClient with AutoCleanableCosmosContainer {
+
+    "spark throughput control" can "use user provided schema" taggedAs RequiresCosmosEndpoint in {
```

Comment:
please modify the test description it should be meaningful. perhaps something like
```scala
"throughput controller" should "support global throughput group" 
```
or something like that

Reply:
changed to  "spark throughput control" should "limit throughput usage"

Reply:
all the test here are within the scope of spark, so I wouldn't use "spark" in the test name.

Reply:
But other test classes have the Spark in the prefix? 
![image](https://user-images.githubusercontent.com/64233642/110397007-e5a6ef00-8025-11eb-9461-a5614d52d42e.png)


Or is the plan we are going to remove it?

Reply:
what you have is good and consistent with others. you are right.

thanks for the response. Please resolve this comment. no action is needed.  :-)